src/rp_turn/tests/test_config_applicator.py

```diff
@@ -656,6 +656,10 @@ def is_settings_valid(self):
         else:
             self.assertNotIn("use-auth-secret", turnconf_file)
 
+        terminal = TestDefaultSettings.DummyTerminal
+        self.assertIn("/usr/bin/chown root:turnserver /etc/turnuserdb.conf", terminal)
+        self.assertIn("/usr/bin/chmod 640 /etc/turnuserdb.conf", terminal)
```

Comment:
if this assert fails, this `os.remove` command is actually run and will try to delete this file on the host system :thinking: I have no idea why it does this

https://github.com/pexip/rp-turn/blob/e63fbf52e2b5b7d32db0360561c64f7fb00a3433/src/rp_turn/config_applicator.py#L278